### PR TITLE
Create rooms and send and display messages

### PR DESCRIPTION
**Pull request by alexrosen45**

### CODE REVIEW

File: src/data_access/FirebaseMessageDataAccessObject.java [hunk 1]
```diff
@@ -19,13 +18,10 @@ public FirebaseMessageDataAccessObject(OkHttpClient client) {
 
   @Override
   public void sendMessage(
-      Room room, LoginUserDataAccessInterface userDAO, User user, Message message) {
+      Room room, LoginUserDataAccessInterface userDAO, User user, String messageBody) {
```
Comment (modderme123) [inline src/data_access/FirebaseMessageDataAccessObject.java:21]: Nice, I prefer this!

**Review by modderme123 — APPROVED** [review]
This seems good!

**Review by uoftm — APPROVED** [review]
Oops, approving with correct account now

File: src/use_case/rooms/LoadRoomsInteractor.java [hunk 2]
```diff
@@ -21,13 +21,15 @@ public LoadRoomsInteractor(
...
   public void loadRooms(LoadRoomsInputData roomsInputData) {
-    // TODO: Trigger this function from the view
     try {
       List<String> availableRoomIds =
           roomsDataAccessObject.getAvailableRoomIds(roomsInputData.user);
       List<Room> rooms = new ArrayList<>();
       for (String roomId : availableRoomIds) {
         Room room = roomsDataAccessObject.getRoomFromId(roomsInputData.user, userDao, roomId);
+        System.out.println(room.getUid());
+        System.out.println(room.getMessages().get(0).content);
```
Comment (uoftm) [inline src/use_case/rooms/LoadRoomsInteractor.java:31]: this is breaking for chats with no messages

**Pull request merged**